### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by HiFaraz** [HiFaraz/atom-starter-pack#1]
@JTronLabs let's discuss here. I've added you as a collaborator if you want to push changes.

Your original comment: https://github.com/HiFaraz/node-playbook/pull/14#issuecomment-243324803

>
Installing your Atom starter pack
sh: 1: installer.sh: not found
npm WARN aaa No description
npm WARN aaa No repository field.
npm WARN aaa No license field.
npm ERR! Linux 4.4.0-34-generic
npm ERR! argv "/usr/bin/nodejs" "/usr/bin/npm" "install" "atom-starter-pack"
npm ERR! node v4.2.6
npm ERR! npm v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
>
npm ERR! atom-starter-pack@0.1.8-alpha preinstall: echo Installing your Atom starter pack && installer.sh
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the atom-starter-pack@0.1.8-alpha preinstall script 'echo Installing your Atom starter pack && installer.sh'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the atom-starter-pack package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR! echo Installing your Atom starter pack && installer.sh
npm ERR! You can get information on how to open an issue for this project with:
npm ERR! npm bugs atom-starter-pack
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! npm owner ls atom-starter-pack
npm ERR! There is likely additional logging output above.
>
npm ERR! Please include the following file with any support request:
npm ERR! /home/james/aaa/npm-debug.log

**Comment by HiFaraz** [comment]
I'm unable to replicate this error. Maybe it's a Linux thing?

Can you try downloading `installer.sh` and then running `echo Installing your Atom starter pack && installer.sh`? That's what the preinstall script is trying to do.

**Comment by JTronLabs** [comment]
What's your package.json for the test project? I am making a new folder, touching package.json, and attempting to run `npm install atom-starter-pack --save-dev`

**Comment by HiFaraz** [comment]
I just installed it in an empty folder. Ideally I'd like it to uninstall itself or delete itself afterwards, but I wasn't having much success.

**Comment by JTronLabs** [comment]
IDK what's going on. Will take another look tomorrow after work. Night!

**Comment by HiFaraz** [comment]
You'll need to publish to npm to test this. What's your npm username? I'll add you as a collaborator there too.

**Comment by JTronLabs** [comment]
https://www.npmjs.com/~jtronlabs

**Comment by HiFaraz** [comment]
You're in!

**Comment by JTronLabs** [comment]
If you're free, maybe try `&& ./installer.sh`. http://stackoverflow.com/questions/34937724/running-bash-scripts-with-npm Can't do it myself quite yet

**Comment by JTronLabs** [comment]
https://github.com/HiFaraz/atom-starter-pack/commit/c1db2c105a9cd64c19020255644ec8dfe64d0725

https://github.com/HiFaraz/atom-starter-pack/commit/3ed6109115a26055be23eb1256b36d72b31b1720

**Comment by HiFaraz** [comment]
Nice! You tested through npm publish and install?

**Comment by JTronLabs** [comment]
tested it locally

http://podefr.tumblr.com/post/30488475488/locally-test-your-npm-modules-without-publishing

**Comment by HiFaraz** [comment]
That's really handy! I'm test this on Windows tonight, and if it works I'll link to it from the playbook.

**Comment by HiFaraz** [comment]
I made some changes and tested it on Windows. Try it now on Linux?

**Comment by HiFaraz** [comment]
@dodekeract would you help us test this using npm pack and npm install for the local tar?

**Comment by HiFaraz** [comment]
Hey @JTronLabs are you able to test this on Linux?

**Comment by dodekeract** [comment]
@HiFaraz No, I need my Atom to be working properly right now, I'd suggest you to try it in a VM.

**Comment by JTronLabs** [comment]
Sorry I forgot! Don't think I'll be able to get it this weekend, I'm going to be away from my dev machine. I can test on Tuesday though
